### PR TITLE
Adding news for 0.5.0

### PR DESCRIPTION
**Pull request by sishida**
Signed-off-by: Sarah Ishida <sishida@us.ibm.com>

PR for https://github.com/eclipse/codewind/issues/647.

### CODE REVIEW

**Comment by sishida** [comment]
Hi, team,
I've created this PR to add a news page for 0.5.0.
- @elsony Could you please review the Che section?
- @jopit Could you please review the IDEs, Eclipse, and VS Code sections?
- @sghung  Could you please review the Appsody, Eclipse, and Website sections?
Thank you!

**Comment by micgibso** [comment]
@elsony Could you review the Che section please?

File: docs/_news/news05.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+---
+
+## Codewind 0.5.0
+Thursday 17 October 2019
+
+#### Appsody
+- [Appsody binding support is available for VS Code and Eclipse.](https://github.com/eclipse/codewind/issues/292)
+
+#### Che
+- Codewind supports the OpenShift project type in Eclipse Che running on OpenShift or OKD. Create and build applications with Codewind and OpenShift.
```
[the remark this answers is not in the document]
Comment (GeekArthur) [reply docs/_news/news05.md]: Submit my review for my previous comment.

[the remark this answers is not in the document]
Comment (GeekArthur) [reply docs/_news/news05.md]: Yea, we can point to Codewind documentation once Jake finishes that documentation.

[the remark this answers is not in the document]
Comment (micgibso) [reply docs/_news/news05.md]: New issue opened for the above, https://github.com/eclipse/codewind/issues/709 
Will get this PR folded in and work the comments from the new issue.

[the remark this answers is not in the document]
Comment (micgibso) [reply docs/_news/news05.md]: These comments now in issue https://github.com/eclipse/codewind/issues/709

**Pull request merged**